core/app/models/spree/customer_return.rb

```diff
@@ -16,9 +16,6 @@ class CustomerReturn < Spree::Base
 
     accepts_nested_attributes_for :return_items
 
-    extend DisplayMoney
-    money_methods pre_tax_total: { currency: Spree::Config[:currency] }
```

Comment:
```ruby
money_methods :pre_tax_total
```

plus 

```ruby
def currency
  order.currency || Store.default.default_currency
end
```

Reply:
Maybe I'll just delegate `currency` without ` || Store.default.default_currency`?
```
# core/app/models/spree/order.rb:723
    def ensure_currency_presence
      self.currency ||= store.default_currency || Spree::Config[:currency]
    end
```

Reply:
@tomurb that will also do

Reply:
@tomurb please also add a test for this feature :)